aws-cloudformation/aws-cloudformation-resource-providers-lookoutvision
# Implement Create/Read Handlers

### Details
* Update .rdpk-config and aws-lookoutvision-project.json schema definition

* Implement Create/Read Handlers

* Use injectCredentialsAndInvokeV2 in create and read handlers

* Increase test coverage

* Run mvn verify in CodeBuild

* Remove unnecessary inheritance and deleted BaseHandlerStd abstract class

* Update Delete, List, and Update to no-op

### Testing
* Added unit tests
* Manually tested via sam
  * See files under `sam-tests` directory. 
  * [Tested following these instructions](https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/resource-type-walkthrough.html#resource-type-walkthrough-test)
  * Verified Project was created via SAM tests
